src/main/kotlin/com/epam/brn/batch/answer/AnswerBatchConfiguration.kt

```diff
@@ -0,0 +1,97 @@
+package com.epam.brn.batch.answer
+
+import com.epam.brn.batch.JobCompletionNotificationListener
+import com.epam.brn.constant.Batch.RESOURCE_ANSWER_INPUT_FILE
+import com.epam.brn.dto.AnswerDto
+import org.apache.commons.lang3.math.NumberUtils
+import org.springframework.batch.core.Job
+import org.springframework.batch.core.Step
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory
+import org.springframework.batch.core.launch.support.RunIdIncrementer
+import org.springframework.batch.item.ItemProcessor
+import org.springframework.batch.item.database.BeanPropertyItemSqlParameterSourceProvider
+import org.springframework.batch.item.database.JdbcBatchItemWriter
+import org.springframework.batch.item.database.builder.JdbcBatchItemWriterBuilder
+import org.springframework.batch.item.file.FlatFileItemReader
+import org.springframework.batch.item.file.LineMapper
+import org.springframework.batch.item.file.mapping.DefaultLineMapper
+import org.springframework.batch.item.file.mapping.FieldSetMapper
+import org.springframework.batch.item.file.transform.DelimitedLineTokenizer
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.core.io.Resource
+import javax.sql.DataSource
+
+@Configuration
+class AnswerBatchConfiguration {
+
+    @Autowired
+    private lateinit var jobBuilderFactory: JobBuilderFactory
+
+    @Autowired
+    private lateinit var stepBuilderFactory: StepBuilderFactory
+
+    @Value(RESOURCE_ANSWER_INPUT_FILE)
+    lateinit var inputResource: Resource
+
+    @Bean
+    fun importAnswersJob(listener: JobCompletionNotificationListener, answerImportFirstStep: Step): Job {
+        return jobBuilderFactory.get("importAnswerJob")
```

Comment:
Is it a typo in input parameter? Am I right that we should use job name "importAnswer**s**Job" instead of "importAnswerJob"?